examples/ola-throughput-multi.cpp

```diff
@@ -0,0 +1,177 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * ola-throughput-multi.cpp
+ * Send a bunch of frames quickly on multiple universes to load test the server.
+ * Copyright (C) 2005 Simon Newton
```

Comment:
Didn't realise you'd change your name @DaAwesomeP ! 😆 

You could always add a line about based on/inspired by and mention the other file if you were really worried, but I'd say there's quite a lot of that already within OLA, e.g. copying plugins etc.